gluten-core/src/main/scala/io/glutenproject/execution/HashJoinExecTransformer.scala

```diff
@@ -966,8 +964,12 @@ trait HashJoinLikeExecTransformer
       .append("buildHashTableId=").append(buildHashTableId).append("\n")
       .append("isExistenceJoin=").append(
       if (joinType.isInstanceOf[ExistenceJoin]) 1 else 0).append("\n")
+    val message = StringValue
+      .newBuilder()
+      .setValue(joinParametersStr.toString)
+      .build()
```

Comment:
@zzcclp In this PR we converted the substrait plan to json on java side, and printed the json string in df.explain(true). The to-json conversion on java side failed in our previous way, and is fixed by using StringValue to pack strings in Any. This also requires each backend to use the `UnpackTo` to parse this field. Could you help update in CH backend? Here's the update in Velox backend: https://github.com/oap-project/velox/pull/72/files#diff-6bbec4a6c456cb1efcf0c8f2dae7b80982e9002b9cf2a3460fd1ae545f291e49

Reply:
@liuneng1994 please help to review and update.